src/harness/fourslash.ts

```diff
@@ -2997,30 +2997,18 @@ Actual: ${stringify(fullActual)}`);
             }
         }
 
-        public verifyCodeFixAvailable(negative: boolean, info: FourSlashInterface.VerifyCodeFixAvailableOptions[] | undefined) {
+        public verifyCodeFixAvailable(info: FourSlashInterface.VerifyCodeFixAvailableOptions[]) {

```

Comment:
I think you also want to move this to explicit ```Verify``` implementation to avoid having it on ```verify.not```
(Move to https://github.com/Microsoft/TypeScript/pull/24325/files#diff-cd92e8371fce86965331cb68f53306f1L4134)

Reply:
Since @amcasey requested it I added back `verify.not`.